saleor/graphql/order/tests/test_order.py

```diff
@@ -450,6 +450,59 @@ def test_draft_order_query(staff_api_client, permission_manage_orders, orders):
     assert len(edges) == Order.objects.drafts().count()
 
 
+def test_order_update_settings(
```

Comment:
We should also add tests for running mutation by the app, and by the user without permission.